cli/command/service/ps.go

```diff
@@ -58,8 +59,11 @@ func runPS(dockerCli *command.DockerCli, opts psOptions) error {
 	serviceIDFilter := filters.NewArgs()
 	serviceNameFilter := filters.NewArgs()
 	for _, service := range opts.services {
+		// default to container runtime
 		serviceIDFilter.Add("id", service)
+		serviceIDFilter.Add("runtime", string(swarmtypes.RuntimeContainer))
 		serviceNameFilter.Add("name", service)
+		serviceNameFilter.Add("runtime", string(swarmtypes.RuntimeContainer))
```

Comment:
I don't believe so.  `Add` adds to a map and it's setting the key `runtime` in addition to `name`.